scripts/Format-Build-Macro.ps1

```diff
@@ -0,0 +1,24 @@
+<#
+  .SYNOPSIS
+  Autoformats `build!` macros across `build.rs` scripts,
+  and `generate!` in bindings' `main.rs`.
+#>
+
+function Format-Macro {
+  param (
+    $File,
+    $Pattern
+  )
+
+  $shim = 'use format_build_macro::{'
+
+  $fmtResult = ((Get-Content -Raw $File).replace($Pattern, $shim) | & rustfmt) -join "`n"
```

Comment:
I'm not too happy with this. It seems the only way to pass _a `String`_ to `stdin` is by using a pipeline, but that automatically splits the output from `rustfmt` into an array of strings by `` `n ``. `Out-String` is encouraged to be used but that adds a trailing newline, uglifying the files and breaking the resulting diff
: https://github.com/PowerShell/PowerShell/issues/14444. All I really want is some equivalent to `-Raw` (don't process the output from the command _at all_) just like on `Get-Content.